src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-panel" data-qa="big">
+      <label for="bigSearch"></label>
+      <img
+        class="icon"
+        src="images/Search.svg"
+        alt="icon1"
+      >
+      <input
+        type="text"
+        id="bigSearch" class="search-bar"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles”"
+      >
+    </form>
+    <form class="search-panel-small" data-qa="small">
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/5af48728-cc96-4ee2-87cd-4a7e70d433ec)


```suggestion
    <form class="search-panel search-panel-small" data-qa="small">
```
in `search-panel-small` only overwrite styles that should be different than in the big panel
do not duplicate the same styles

in other words `search-panel` will be basic